Apps/WebClient/src/ClientApp/src/components/report/ImmunizationHistoryReportComponent.vue

```diff
@@ -257,7 +261,13 @@ export default class ImmunizationHistoryReportComponent extends Vue {
 
 <template>
     <div>
-        <section>
+        <div
+            v-if="isRecommendationEmpty && isEmpty && !isLoading"
+            class="mt-2 d-md-none"
+        >
+            No records found.
+        </div>
+        <section v-else-if="!isDependent">
```

Comment:
I think you can simplify the display logic now by adding the `d-none d-md-block` classes to this element and removing all of the `d-none d-md-block` and `d-none d-md-table` classes from its descendants.